src/language-handlebars/printer-glimmer.js

```diff
@@ -205,7 +205,7 @@ function print(path, options, print) {
               softline,
               path
                 .map(partPath => print(partPath), "parts")
-                .filter(a => a !== "")
+                .filter(a => !isWhitespaceText(a))
```

Comment:
I don't this this is correct, with this change the following is happening:

```
<div class="{{hello}} {{world}}"></div>
``` 

=> 

```
<div class="{{hello}}{{world}}"></div>
```